server/resource_controller.go

```diff
@@ -266,3 +270,34 @@ func responseURL(r *http.Request, config Config, paths ...string) *url.URL {
 
 	return &responseURL
 }
+
+// UnescapedJSONRenderer replaces gin's default JSON renderer and ensures
+// that the special characters "<", ">", and "&" are not escaped after the
+// the JSON is marshaled. Escaping these special HTML characters is the default
+// behavior of Go's json.Marshal().
+type UnescapedJSONRenderer struct {
+	obj interface{}
+}
+
+func (u UnescapedJSONRenderer) Render(w http.ResponseWriter) (err error) {
+	data, err := json.Marshal(&u.obj)
+	if err != nil {
+		return
+	}
+
+	// Replace the escaped characters in the data
+	data = bytes.Replace(data, []byte("\\u003c"), []byte("<"), -1)
+	data = bytes.Replace(data, []byte("\\u003e"), []byte(">"), -1)
+	data = bytes.Replace(data, []byte("\\u0026"), []byte("&"), -1)
```

Comment:
Man, this stinks.  It really is too bad that the other approach did not work...  I wonder why... Anyway, cheers for getting *something* that works.  I hope it isn't too much of a performance hit!